### PR TITLE
Upgrade buttons for setGuestUsername

### LINKED ISSUE #5500: Bogus translation
https://github.com/nextcloud/spreed/blob/3606f14153d956d5513c2646d45a172bc0517376/src/components/SetGuestUsername.vue#L28

The trailing space is missing in all translations, so instead a placeholder needs to me used

### PR DESCRIPTION
**Pull request by marcoambrosini**
Screenshot 2022-01-21 at 15 20 32

Fix #5500

### CODE REVIEW

File: src/components/SetGuestUsername.vue [hunk 1]
```diff
@@ -26,12 +26,16 @@
 		@submit.prevent="handleChooseUserName">
 		<h3>
 			{{ t('spreed', 'Display name: ') }} <strong>{{ actorDisplayName ? actorDisplayName : t('spreed', 'Guest') }}</strong>
-			<button
-				class="icon-rename"
-				@click.prevent="handleEditUsername">
-				{{ t('spreed', 'Edit') }}
-			</button>
 		</h3>
+		<Button type @click.prevent="handleEditUsername">
```
Comment (nickvergessen) [inline src/components/SetGuestUsername.vue:30]: ???

**Comment by nickvergessen** [comment]
Before | After
---|---
Bildschirmfoto von 2022-01-25 09-30-43 | Bildschirmfoto von 2022-01-25 09-30-18

Not sure I like the new version better.
Basically all 3 texts "Display name", "Edit" and "Settings" start on a different offset which feels weird.
Either we inline it again, or make settings a button and put it in a line with the edit button.
Also since the edit button is not on the name anymore, the label should be "Edit name"?

**Comment by nickvergessen** [comment]
Conflicting files
src/components/SetGuestUsername.vue

**Comment by nikola-gladovic** [comment]
@marcoambrosini 
Could you please fix this conflicts?
Or shall I do this? :)

**Comment by nickvergessen** [comment]
Bump

**Review by nickvergessen — APPROVED** [review]

**Comment by nickvergessen** [comment]
@nikola-gladovic review? :)

**Pull request merged**